server/controllers/api/videos/index.ts

```diff
@@ -184,17 +203,38 @@ async function addVideo (req: express.Request, res: express.Response) {
 
   const videoPhysicalFile = req.files['videofile'][0]
   const videoInfo: VideoCreate = req.body
+  const files = req.files
+
+  return addVideo(req, res, { videoPhysicalFile, videoInfo, files })
+}
 
-  const videoData = buildLocalVideoFromReq(videoInfo, res.locals.videoChannel.id)
+async function addVideoResumable (req: express.Request, res: express.Response) {
+  const videoPhysicalFile = res.locals.videoFileResumable
+  const videoInfo = videoPhysicalFile.metadata
+  const files = { image: { path: getResumableUploadPath(videoPhysicalFile.filename) } }
+
+  return addVideo(req, res, { videoPhysicalFile, videoInfo, files })
+}
+
+async function addVideo (req: express.Request, res: express.Response, parameters: {
+  videoPhysicalFile: { duration: number, filename: string, size: number, path: string }
+  videoInfo: VideoCreate | Metadata
+  files
```

Comment:
Could you type this?

Reply:
It was hard to find a good name. I went for `UploadVideoFiles`, but feel free to propose something better.